### PR TITLE
Landing

### PR DESCRIPTION
**Pull request by asvae**

### CODE REVIEW

File: packages/docs/assets/colors.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+$active-blue: #2550C0;
```
Comment (asvae) [inline packages/docs/assets/colors.scss:1]: Routes are not right:
* `/en/landing` replace with `/en`.

Comment (asvae) [inline packages/docs/assets/colors.scss:1]: Language switching doesn't seem to work for me
image

File: packages/docs/components/landing/Admin.vue [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+          <va-button class="admin__buttons--button"
+            href="https://vuestic.epicmax.co/admin/dashboard"
+            :round="false"
+            color="#2550C0"
+            flat
+          >
+            {{$t('landing.admin.buttons.demo')}}
+          </va-button>
+          <va-button-group class="admin__buttons--button">
+            <va-button class="star-button" color="#f4f4f4" href="https://github.com/epicmaxco/vuestic-admin" :round="false" size="small">
```
Comment (asvae) [inline packages/docs/components/landing/Admin.vue:25]: image
Move closer

image
Same here.
Both low priority.

File: packages/docs/assets/colors.scss (hunk 1 above)
Comment (asvae) [inline packages/docs/assets/colors.scss:1]: Can we do a couple of px of padding between buttons?
image

Comment (asvae) [inline packages/docs/assets/colors.scss:1]: Styling as different frameworks - either split as separate task or finish in context of this MR.
image

File: packages/docs/components/landing/Customize.vue [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+      color: #ffffff;
+      padding-top: 1rem;
+      line-height: 1.5rem;
+    }
+  }
+}
+
+.block {
+  &__components {
+    @include row-flex();
```
Comment (asvae) [inline packages/docs/components/landing/Customize.vue:239]: Can you check if we can use vuestic grids, and if yes, can we use them for simple cases at least?

**Review by asvae — COMMENTED** [review]
Looks good

File: packages/docs/assets/colors.scss (hunk 1 above)
Comment (Kushich) [reply packages/docs/assets/colors.scss:1]: It's because we don't have Russian translation yet

**Pull request merged**